### ISSUE TITLE
CSAS Error message missing space and ambiguous

### ISSUE THREAD

**Issue by rmoff** [confluentinc/ksql#990]
```
ksql> CREATE STREAM TARGET_TOPIC WITH (PARTITIONS=1, REPLICAS=3) AS \
> SELECT * FROM SOURCE_TOPIC;
io.confluent.ksql.exception.KafkaResponseGetFailedException: Failed to guarantee existence of topicTARGET_TOPIC
ksql>
```

Above error has two issues: 
1. There's a missing space between `topic` and the topic name (`topicTARGET_TOPIC` should read `topic TARGET_TOPIC`)
2. It's unclear if this is a fatal error or just a warning. Just because something's not been guaranteed, it doesn't mean to me that it's not happened - just that we're not quite sure. I would change this message to read something like "Failed to create topic <foo>" and then ideally supply the reason too.

**Comment by big-andy-coates** [comment]
Hey @rmoff,

1. I think this has been fixed in 4.1.x already. What version are you checking?

2. The the topic may or may not already exist, so we're no trying to create the topic, we're trying to ensure it exists and has the right config.  If the topic exists, but there was a network blip or ACLs stopped us seeing that it exists, then a message of "Failed to create topic" might mislead users, so I'm not sure its better.

**Comment by rmoff** [comment]
1. confluent-4.1.0-SNAPSHOT nightly build as of 12th March
2. I understand what you're saying. As an end user though, I'm left
unsure…do I plough on regardless? Do I need to retry?

**Comment by big-andy-coates** [comment]
What to do, what to do.... :D

1. Code now as a space and was changed on the 23rd of Feb in 4.1.x branch. :-/
  - master branch still has issue: https://github.com/confluentinc/ksql/blob/master/ksql-engine/src/main/java/io/confluent/ksql/util/KafkaTopicClientImpl.java#L101
  - 4.1.x branch does not: https://github.com/confluentinc/ksql/blob/4.1.x/ksql-engine/src/main/java/io/confluent/ksql/util/KafkaTopicClientImpl.java#L101

2. Maybe the message should say something like:

> Failed to guarantee existence of topic BLAH. Consider dropping the stream or table and recreating.

And maybe include a stack trace for more info? (Not sure what our view is about showing stack traces in the CLI?)

**Comment by apurvam** [comment]
this seems to be fixed on 4.1.x and master. Can you reproduce this with a newer nightly?

**Comment by rmoff** [comment]
Cannot reproduce. Closing.
